paasta_tools/cli/cmds/check.py

```diff
@@ -202,44 +203,34 @@ def yaml_check(service_path):
         paasta_print(PaastaCheckMessages.YAML_MISSING)
 
 
-def get_chronos_steps(service, soa_dir):
-    """This is a kind of funny function that gets all the chronos instances
-    for a service and massages it into a form that matches up with what
+def get_framework_steps(framework, service, soa_dir):
```

Comment:
This will definitely make this function look better, however then we will subtract deploy_groups from steps in deployments_check().